test/e2e/storage/empty_dir_wrapper.go

```diff
@@ -172,7 +176,13 @@ var _ = utils.SIGDescribe("EmptyDir wrapper volumes", func() {
 	// but these cases are harder because tmpfs-based emptyDir
 	// appears to be less prone to the race problem.
 
-	It("should not cause race condition when used for configmaps [Serial] [Slow]", func() {
+	// SLow by design [~180 Seconds]
```

Comment:
Let's put this in the Description field instead

Reply:
Done.